src/components/Background/planetoids/Planet.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Station.css';
+
+const stationHeight = 27;
+
+function Planet(props) {
+    return (
+        <div style={{
```

Comment:
I guess this is probably a necessity of the react setup, but having all this inline styling is pretty rough haha. This doesn't apply so much for stuff like planet which are one-use, but for other stuff I think it'd be a lot more manageable if you can keep as much stuff as possible in stylesheets.

Reply:
I don't disagree -- I think this is something I think will need to be in a follow on PR. The standard create-react-app setup doesn't put in a preference for styling tools. Plain CSS (which I've used in some places) is not a great long term solution. CSS-modules or styled components would be a preferred solution, but I didn't go through the exercise of setting those up yet. Even just a simple SCSS replacement would be an improvement.

For this PR I tried to keep the CSS usage limited to things that were used by multiple elements, like list-items or what-have-yous. The inline styles do have an advantage of not leaking, which is somewhat useful in a proof-of-concept type project like this.

Reply:
That all sounds good. As I've said enough times, I don't know react at all so do what makes sense and take what I say with a lot of grains of salt haha.
I'd agree the cleanup for this should be done in future work sometime, no need for it to hold up this PR. Beyond that, yeah, I'm cool with whatever of those options you like the most/think would be best.